decidim-admin/app/commands/decidim/admin/create_scope.rb

```diff
@@ -3,49 +3,32 @@
 module Decidim
   module Admin
     # A command with all the business logic when creating a static scope.
-    class CreateScope < Decidim::Command
+    class CreateScope < Decidim::Commands::CreateResource
+      fetch_form_attributes :name, :organization, :code, :scope_type
       # Public: Initializes the command.
       #
       # form - A form object with the params.
       # parent_scope - A parent scope for the scope to be created
```

Comment:
If we're changing the instance variable from `@parent_scope` to `@parent`, I think we should change it also in the other places, to prevent bugs as the one in line 28 (calling an non existing instance variable)

```suggestion
      # parent - A parent scope for the scope to be created
```